### PR TITLE
ipv6calc 4.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
Can't locate URI/Escape.pm in @INC (you may need to install the URI::Escape module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.22.1 /usr/local/share/perl/5.22.1 /usr/lib/x86_64-linux-gnu/perl5/5.22 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.22 /usr/share/perl/5.22 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at ./ipv6calcweb.cgi line 70.
BEGIN failed--compilation aborted at ./ipv6calcweb.cgi line 70.
make[4]: Leaving directory '/tmp/ipv6calc-20211110-2684-mrxrgu/ipv6calc-4.0.0/databases/lib'
make[3]: Leaving directory '/tmp/ipv6calc-20211110-2684-mrxrgu/ipv6calc-4.0.0/databases/lib'
Makefile:13: recipe for target 'ipv6calcweb' failed
make[1]: *** [ipv6calcweb] Error 2
```

linux build failure

**Comment by iMichka** [comment]
```
Can't locate URI/Escape.pm in @INC (you may need to install the URI::Escape module) (@INC contains: /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0 /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/5.34.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.34.0/lib/perl5/5.34.0 /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.34.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.34.0) at ./ipv6calcweb.cgi line 70.
```

Ha ha got confused by this, as we do set PERL5LIB correctly just above. It's a `cgi` script and it does not care about PERL5LIB ...

**Comment by iMichka** [comment]
Got it to work. Don't know if we can do something better for that cgi script though ...

File: Formula/ipv6calc.rb [hunk 1]
```diff
@@ -15,7 +19,52 @@ class Ipv6calc < Formula
...
+
+      resources.each do |r|
+        r.stage do
+          system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}"
+          system "make", "install"
+        end
+      end
+    end
+
+    if OS.linux?
```
Comment (carlocab) [inline Formula/ipv6calc.rb:58]:
```suggestion

```

File: Formula/ipv6calc.rb [hunk 2]
```diff
@@ -15,7 +19,52 @@ class Ipv6calc < Formula
...
+      sha256 "64d9e2eb2b420f1492da01ec0e6976363245b4be9290f03f10b7d2cb63fa2f61"
+    end
+  end
+
+  resource "DIGEST::Sha1" do
+    on_linux do
+      url "https://cpan.metacpan.org/authors/id/G/GA/GAAS/Digest-SHA1-2.13.tar.gz"
+      sha256 "68c1dac2187421f0eb7abf71452a06f190181b8fc4b28ededf5b90296fb943cc"
+    end
+  end
```
Comment (carlocab) [inline Formula/ipv6calc.rb:43]: Can we not put `resource` blocks in `on_linux` blocks, so that we only have one `on_linux` block instead of three?

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**